components/Card.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Link from 'next/link';
+import TextTruncate from 'react-text-truncate';
+interface CardProps {
+  title: string;
+  body: string;
+  icon?: string;
+  link?: string;
+  image?: string;
+}
+
+const CardBody = ({ title, body, icon, link, image }: CardProps) => {
+  return (
+    <div className='group relative h-full w-full max-w-lg rounded-lg border border-gray-200 bg-white p-6 px-12 shadow-3xl transition-colors delay-[150ms] ease-in-out hover:bg-slate-100'>
```

Comment:
@TamannaVerma99 Please add `dark:text-black` in the div. As in dark mode the text is not visible.
![image](https://github.com/json-schema-org/website/assets/96608169/b8ddf834-8ffb-4771-9948-311ac746f215)